src/step.rs

```diff
@@ -31,6 +32,28 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
         self.memory.clear_packed();
         self.inc_step_counter_and_check_limit(1)?;
         if self.stack.is_empty() {
+            if let Some((instance, ptr)) = self.memory.fetch_tls_dtor() {
+                trace!("Running TLS dtor {:?} on {:?}", instance, ptr);
+                // TODO: Potientiually, this has to support all the other possible instances? See eval_fn_call in terminator/mod.rs
+                let mir = self.load_mir(instance.def)?;
+                // FIXME: Are these the right dummy values?
```

Comment:
Well I kind of considered the return pointer also a dummy value (as the function should not return anything), and I have no idea what the `StackPopCleanup` thing does, so I referred to them as dummys as well.

Are you saying that I should get the span from the `instance`, and then nothing is actually a dummy value? Not sure how I would get that span though.